app/boards/shields/bfo9000/bfo9000_left.conf

```diff
@@ -0,0 +1,2 @@
+CONFIG_ZMK_SPLIT=y
+CONFIG_ZMK_SPLIT_BLE_ROLE_PERIPHERAL=y
```

Comment:
Any particular reason you've chosen the left as peripheral and right as central for this board? It's a bit out of the norm with the rest of the shields.

Reply:
I had built a right first, that's all. It's really arbitrary with these boards. Happy to change if you prefer.

Reply:
I would prefer it's switched for the reason that when people come for support, we assume that the left side is central. Since it's otherwise arbitrary, this should make support more straight forward if every board has the left side as central by default.

Reply:
Done!